packages/generation-runner/src/react/contexts/generation-runner-system.tsx

```diff
@@ -304,6 +305,13 @@ export function GenerationRunnerSystemProvider({
 		[client],
 	);
 
+	const callGithubOperation = useCallback(
+		async (generation: QueuedGeneration) => {
+			await client.executeGitHubOperation({ generation });
+		},
+		[client],
+	);
+
```

Comment:
since generation-runner-system is system for runners, I don't add github specific process to this provider.